docs/io/unified.rst

```diff
@@ -378,6 +386,13 @@ in the format of their choosing.
     >>> tm['a'] == t['a'].value
     array([ True,  True], dtype=bool)
 
+This write behaviour of Time will be useful when dealing with other software packages
```

Comment:
I think you just explained that above, so would suggest deleting this paragraph.